### PR TITLE
Update app-protection-policy-settings-ios.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note stating that App protection policy will be requited with IntuneMAMUPN for Managed Devices.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger12** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/1722

File: memdocs/intune/apps/app-protection-policy-settings-ios.md [hunk 1]
```diff
@@ -61,6 +61,9 @@ There are three categories of policy settings: *Data relocation*, *Access requir
...
+  
+> [!NOTE]
+> App protection policy will be requited with IntuneMAMUPN for Managed Devices.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/app-protection-policy-settings-ios.md:66]:
```suggestion
> An app protection policy is required with IntuneMAMUPN for managed devices.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger4** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please review and merge this PR. Thanks

**Comment by yogkumgit** [comment]
@Erikre Gentle reminder, Please merge this PR. Thanks

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**